api/helm-app/HelmAppRestHandler.go

```diff
@@ -304,7 +309,7 @@ func (handler *HelmAppRestHandlerImpl) DeleteApplication(w http.ResponseWriter,
 		return
 	}
 
-	res, err := handler.helmAppService.DeleteApplication(r.Context(), appIdentifier)
+	res, err := handler.helmAppService.DeleteBaseStackHelmApplication(r.Context(), appIdentifier, userId)
```

Comment:
DeleteBaseStackHelmApplication can be renamed

Reply:
fixed 👍